File: Tasks/difference.py
def diff(object_1,object_2):
    if object_1 == object_2:
        return False
    for attribute_name in object_2:
        object_1[attribute_name]=object_2[attribute_name]
        del object_1[attribute_name]
    return object_1
result = diff({'a':'uno', 'b':'due'},{'a':'uno', 'c':'tre'})
print(result)

File: Tasks_edited/difference.js
'use strict';
const diff = (dictionary1, dictionary2) => {
  const differ = {};
  for (const index of Object.keys(dictionary1)) {
    differ[index] = dictionary1[index];
  }
  for (const index of Object.keys(dictionary2)) {
    if  (dictionary1[index] === dictionary2[index]) {
      delete differ[index];
    }
  }
  return dictionary2;
};
const result = diff({ a: 'uno', b: 'due' }, { a: 'uno', c: 'tre' });
console.log(result);
